ui/app/appointments/controllers/manage/list/appointmentsListViewController.js

```diff
@@ -199,6 +209,7 @@ angular.module('bahmni.appointments')
             var showPopUp = function (popUpScope) {
                 popUpScope.no = function (closeConfirmBox) {
                     closeConfirmBox();
+                    console.log($stateParams.viewDate);
```

Comment:
Hi @smartyrad,
Please remove `console.log`

Reply:
Hi @sumanmaity112 , sure. But this is not the final PR. I needed some help in reviewing the code to decide the course of action to be taken to fix this

Reply:
@smartyrad we are looking to do the 0.91 release very soon. Does this PR solve the problem, or not yet?

Reply:
@djazayeri Yes, this issue is almost fixed. But due to a particular test failing, I had listed down a few steps that I could possibly take to solve the issue. Can you please help me out with this?